kalite/settings/base.py

```diff
@@ -188,16 +188,15 @@
 # the user running kalite and should be in a user-data
 # storage place.
 
-USER_WRITABLE_LOCALE_DIR = os.path.join(USER_DATA_ROOT, 'locale')
-KALITE_APP_LOCALE_DIR = os.path.join(USER_DATA_ROOT, 'locale')
+from fle_utils.general import ensure_dir
+USER_WRITABLE_LOCALE_DIR = os.path.join(USER_DATA_ROOT, 'project', 'locale')
+KALITE_APP_LOCALE_DIR = os.path.join(USER_DATA_ROOT, 'project', 'locale')
 
 LOCALE_PATHS = (USER_WRITABLE_LOCALE_DIR, KALITE_APP_LOCALE_DIR)
-if not os.path.exists(USER_WRITABLE_LOCALE_DIR):
-    os.mkdir(USER_WRITABLE_LOCALE_DIR)
-
 DEFAULT_DATABASE_DIR = os.path.join(USER_DATA_ROOT, "database",)
-if not os.path.exists(DEFAULT_DATABASE_DIR):
-    os.mkdir(DEFAULT_DATABASE_DIR)
```

Comment:
I prefer this: `USER_DATA_ROOT` has to exist at this point. If it doesn't, the above breaks.